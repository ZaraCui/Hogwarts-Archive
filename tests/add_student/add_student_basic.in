ADD STUDENT Harry Potter
ADD STUDENT Hermione Granger
ADD STUDENT Ron Weasley
EXIT

