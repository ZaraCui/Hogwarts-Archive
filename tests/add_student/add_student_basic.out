user: Success.

user: Success.

user: Success.

user: Ending Archive process.
